exp/runtime/hooks/api/v1alpha1/lifecyclehooks_types.go

```diff
@@ -180,38 +180,77 @@ func BeforeClusterDelete(*BeforeClusterDeleteRequest, *BeforeClusterDeleteRespon
 
 func init() {
 	catalogBuilder.RegisterHook(BeforeClusterCreate, &runtimecatalog.HookMeta{
-		Tags:        []string{"Lifecycle Hooks"},
-		Summary:     "Called before Cluster topology is created.",
-		Description: "This blocking hook is called after the Cluster is created by the user and immediately before all the objects which are part of a Cluster topology are going to be created.",
+		Tags:    []string{"Lifecycle Hooks"},
+		Summary: "Cluster API Runtime will call this hook before Cluster's topology is created.",
+		Description: "Cluster API Runtime will call this hook after the Cluster is created by the user and immediately before " +
+			"all the objects which are part of a Cluster's topology are going to be created.\n" +
+			"\n" +
+			"Notes:\n" +
+			"- This hook will be called only for Clusters with a managed topology\n" +
+			"- The call's request contains the Cluster object\n" +
+			"- This is a blocking hook; Runtime Extension implementers can use this hook to execute\n" +
+			"tasks before the objects which are part of a Cluster's topology are created",
 	})
 
 	catalogBuilder.RegisterHook(AfterControlPlaneInitialized, &runtimecatalog.HookMeta{
-		Tags:        []string{"Lifecycle Hooks"},
-		Summary:     "Called after the Control Plane is initialized for the first time.",
-		Description: "This non-blocking hook is called after the Control Plane for the Cluster is reachable for the first time.",
+		Tags:    []string{"Lifecycle Hooks"},
+		Summary: "Cluster API Runtime will call this hook after the Control Plane is initialized for the first time",
```

Comment:
```suggestion
		Summary: "Cluster API Runtime will call this hook after the Control Plane has been initialized.",
```
nit: up to you. Initialized just sounds like something which happens only once anyway

Reply:
Maybe we should also just write "after the Control Plane is reachable for the first time"? (like in the description)

Might be easier to understand then "initialized" for most users